integration_test/mysql_mysql_test.go

```diff
@@ -613,3 +613,69 @@ func TestTagDDL(t *testing.T) {
 	err = row.Scan(&tblName)
 	r.Equal(sql.ErrNoRows, err)
 }
+
+func TestDDLNoRoute(t *testing.T) {
```

Comment:
What's the expectation of this test?

When no routes configured, and source executed DDL, the server does  not crash?

Reply:
yes